cloud/aws/templates/aws_oidc/main.tf

```diff
@@ -40,6 +40,12 @@ resource "aws_apprunner_service" "civiform_dev" {
           STAGING_ADMIN_LIST     = var.staging_program_admin_notification_mailing_list
           STAGING_TI_LIST        = var.staging_ti_notification_mailing_list
           STAGING_APPLICANT_LIST = var.staging_applicant_notification_mailing_list
+
+          APPLICANT_OIDC_PROVICER_NAME = var.applicant_oidc_provider_name
```

Comment:
We'll eventually want all the possible variables to configure. Except CLIENT_ID, CLIENT_SECRET, and DISCOVERY_URI, they are optional (with sane defaults)
APPLICANT_OIDC_PROVICER_NAME
APPLICANT_OIDC_CLIENT_ID
APPLICANT_OIDC_CLIENT_SECRET
APPLICANT_OIDC_DISCOVERY_URI
APPLICANT_OIDC_RESPONSE_MODE
APPLICANT_OIDC_RESPONSE_TYPE
APPLICANT_OIDC_ADDITIONAL_SCOPES
APPLICANT_OIDC_LOCALE_ATTRIBUTE
APPLICANT_OIDC_EMAIL_ATTRIBUTE
APPLICANT_OIDC_FIRST_NAME_ATTRIBUTE
APPLICANT_OIDC_MIDDLE_NAME_ATTRIBUTE
APPLICANT_OIDC_LAST_NAME_ATTRIBUTE